examples/graph_rag/graph_knowledge_example.py

```diff
@@ -0,0 +1,623 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please move this to `examples/functionality/graph_rag/`, and translate the comments to English.
Please also add an English `README.md`.